components/overview/index.js

```diff
@@ -278,7 +278,11 @@ const Overview = () => {
 				</span>
 			</div>
 		</div>
-	) : isNil(allNominationsData) && isNil(userData) ? (
+	) : isNil(allNominationsData) &&
+	  isNil(userData) &&
+	  unbondingBalances.length == 0 &&
+	  isNil(unlockingBalances) &&
+	  redeemableBalance == 0 ? (
```

Comment:
Request to change `redeemableBalance == 0` to `redeemableBalance === 0`

Reply:
:)